core/src/main/java/org/testcontainers/containers/ContainerState.java

```diff
@@ -27,14 +29,41 @@ default String getContainerIpAddress() {
     /**
      * @return is the container currently running?
      */
-    default Boolean isRunning() {
+    default boolean isRunning() {
+        if (getContainerId() == null) {
+            return false;
+        }
+
+        try {
+            Boolean running = getCurrentContainerInfo().getState().getRunning();
+            return running != null && running; // avoid NPE when unboxing
```

Comment:
`Boolean.TRUE.equals(running)` ;)

Reply:
Thx, I think it's obvious that I'm not used to working with boxed primitives :D